pom.xml

```diff
@@ -43,6 +42,11 @@
       <artifactId>xercesImpl</artifactId>
       <version>2.11.0</version>
     </dependency>
+    <dependency><!-- 2.4 is required. Avoid transitive import derived issues -->
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+      <version>2.4</version>
```

Comment:
For example here: https://github.com/jenkinsci/analysis-core-plugin/blob/master/src/main/java/hudson/plugins/analysis/core/AbstractAnnotationParser.java#L11

I don't remember exactly, but I think something related to ```org.apache.commons.io.Charsets``` is used somewhere that required 2.4 (it was there before my changes, I only added a explicit dependency to avoid issues). I mean, If you remove this dependency and run ```mvn dependency:tree``` you'll get it there. So this change is really not adding something new.